contrib/guix/README.md

```diff
@@ -9,79 +11,835 @@ downloads.
+documented [here](), but I would recommend that you install v1.2.1 (planned for
+release on April 18th, 2021) or later if your choice of installation option
+allows.
+
+This only needs to be done once per machine. If you have already completed the
+installation and setup, please proceed to [perform a build](#usage).
+
+Otherwise, you may choose from one of the following options to install Guix:
+
+1. Using the official **shell installer script** [⤓ skip to section]()
+   - Maintained by Guix developers
+   - Easiest (automatically performs *most* setup)
+   - Works on nearly all Linux distributions
+   - Only installs latest release
+   - Binary installation only, requires high level of trust
+   - Note: The script needs to be run as root, so it should be inspected before it's run
+2. Using the official **binary tarball** [⤓ skip to section]()
+   - Maintained by Guix developers
+   - Normal difficulty (full manual setup required)
+   - Works on nearly all Linux distributions
+   - Installs any release
+   - Binary installation only, requires high level of trust
+3. Using fanquake's **Docker image** [↗︎ external instructions]()
+   - Maintained by fanquake
+   - Easy (automatically performs *some* setup)
+   - Works on most Linux distributions
+   - Installs any release
+   - Binary installation only, requires high level of trust
+4. Using a **distribution-maintained package** [⤓ skip to section]()
+   - Maintained by distribution's Guix package maintainer
+   - Normal difficulty (manual setup required)
+   - Works only on distributions with Guix packaged, at time of writing:
+     - Debian Bullseye or later
+     - Ubuntu Hirsute or later
+     - Arch Linux (via AUR)
+   - Installs a release decided on by package maintainer
+   - Source or binary installation depending on distribution
+5. Building **from source** [⤓ skip to section]()
+   - Maintained by you
+   - Hard, but rewarding
+   - Can be made to work on most Linux distributions
+   - Installs any commit (more granular)
+   - Source installation, requires lower level of trust
+
+## Option 1: Using the official shell installer script
+
+The shell installer script provided by the Guix maintainers automates the
+download, installation, and initial configuration steps detailed
+[here](https://guix.gnu.org/manual/en/html_node/Binary-Installation.html).
+
+To install Guix using this method, do the following as `root` (e.g. via `su -`):
+
+```sh
+# Use /tmp as the working directory
+cd /tmp
+
+# Download the script
+wget https://git.savannah.gnu.org/cgit/guix.git/plain/etc/guix-install.sh
+
+# Inspect the script
+less guix-install.sh
+
+# Make the script executable
+chmod +x guix-install.sh
+
+# Run the script
+./guix-install.sh
+```
+
+
+## Option 2: Using the official binary tarball
+
+Follow the binary installation instructions step-by-step in the Guix Reference
+Manual
+[here](https://guix.gnu.org/manual/en/html_node/Binary-Installation.html).
```

Comment:
This link contains the instructions that are found in section `Option 1: Using the official shell installer script`. 

Seems as though this section and the first (Option 1 & 2) can be condensed in some way.

For `Option 1`, i don't like that we are re-writing the instructions when we could just point to this section of the mentioned link:
![guix-install-script](https://user-images.githubusercontent.com/23396902/124206049-87a88680-dab0-11eb-865b-aa4d0bbc3f11.png)

Reply:
You're right, I'm not too happy with the current situation either... Will fix! Clearer = better